src/main/java/Deadlines.java

```diff
@@ -0,0 +1,12 @@
+public class Deadlines extends Task{
+    private String by;
+    public Deadlines(String description, String by) {
+        super(description);
+        this.by = by;
+        this.type = TaskType.DEADLINE;
+    }
+    @Override
+    public String getDescription(){
+        return super.getDescription() + "(by: "+ by +")";
+    }
+}
```

Comment:
LGTM

Reply:
yes! I changed the getDescription method to toString method(override from task.java)